### PR TITLE
Feature/ERC-17-aprendamos-categorias

### PR DESCRIPTION
**Pull request by sofiaperezm**
# FooCamp - Escuela de Robotica

## Trello ticket
[User Story 17: Aprendamos_Categorías](https://trello.com/c/7AodzNxW/30-user-story-17-aprendamoscategor%C3%ADas)

## Description
This PR contains:

The `templates/letsLearnCards.hbs` file for the content container in the sub-page _Aprendamos_.
The `templates/letsLearnCard.hbs` file for the single card layout.
The `src/scss/layout/_lets-learn.scss` file for the styles in the content container in the sub-page _Aprendamos_.

## To reproduce

1. Run the project
2. Open the let's learn (http://localhost:8080/aprendamos-page.html) page.
3. Check the component

## Screenshots
The _Aprendamos Categorías_ component in the default size (based on Figma design)
Captura de Pantalla 2021-09-27 a la(s) 1 10 56 a  m

The _Aprendamos Categorías_ component in a medium screen
Captura de Pantalla 2021-09-27 a la(s) 1 12 25 a  m

The _Aprendamos Categorías_ component in a small screen
Captura de Pantalla 2021-09-27 a la(s) 1 12 54 a  m

The filter by category feature in action (sidebar)

https://user-images.githubusercontent.com/32691511/136323301-50a26c89-2204-4532-b643-86918d0ac823.mp4

The filter by category feature in action (dropdown)

https://user-images.githubusercontent.com/32691511/136323162-4571de3a-d77c-49e1-892f-ed24bc3d8392.mp4

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 // Components
 @import 'components/component';
+@import 'components/lets-learn'
```
Comment (diegoamt) [inline src/scss/main.scss]: missing semicolon at the end `;`

File: templates/page.hbs [hunk 2]
```diff
@@ -10,6 +10,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
```
Comment (diegoamt) [inline templates/page.hbs:13]: why do you need to include this fonts ?

**Review by diegoamt — CHANGES_REQUESTED** [review]

File: templates/page.hbs (hunk 2 above)
Comment (sofiaperezm) [reply templates/page.hbs:13]: Hi Diego, I needed it in order to add an icon. I asked @camilomejiag for help on that element.

File: src/scss/main.scss (hunk 1 above)
Comment (sofiaperezm) [reply src/scss/main.scss]: done :)

File: src/scss/abstracts/_variables.scss [hunk 3]
```diff
@@ -2,23 +2,10 @@
   --black: #000000;
   --red: #e60012;
   --white: #ffffff;
-  --bluetitle:#577598;
```
Comment (camilomejiag) [inline src/scss/abstracts/_variables.scss]: Are you sure all the variables you removed here, are not being used anywhere else?

**Review by camilomejiag — COMMENTED** [review]
Sofi, veo que estas eliminando varios de los templates de la app. Estan al final del MR, es por algun motivo?

**Review by camilomejiag — APPROVED** [review]

**Pull request merged**